platform/ios/scripts/package.sh

```diff
@@ -5,9 +5,9 @@ set -o pipefail
 set -u
 
 NAME=Mapbox
-OUTPUT=build/ios/pkg
+OUTPUT=platform/ios/pkg
```

Comment:
@kkaefer, were you planning to change this back to build/ios/pkg/, or were you planning to change [this line](https://github.com/mapbox/mapbox-gl-native/pull/5359/commits/480869cfcad7368b95f17c9f59e8ef008053fe1b#diff-4b90a74e5e02a5a09ced6c70a733a0feR8) to match?